azkaban-common/src/main/java/azkaban/utils/SystemMemoryInfo.java

```diff
@@ -22,34 +17,46 @@
  * All the memory size used in this function is in KB
  */
 public class SystemMemoryInfo {
-  private static final Logger logger = Logger.getLogger(SystemMemoryInfo.class);
+  private static final org.slf4j.Logger logger = LoggerFactory.getLogger(SystemMemoryInfo.class);
```

Comment:
Having this change incrementally can create confusion. I am not sure whether this helps us migrate from one to another. Rather it may lead to a codebase with both loggers in use.